### PR TITLE
cluster import flow should match with UX design

### PR DESCRIPTION
**Pull request by kamleshverma1**
- worked on jira ticket - TEN-171

Signed-off-by: kamleshKumarVerma <kaverma@redhat.com>

### CODE REVIEW

**Comment by kamleshverma1** [comment]
@gnehapk Please review it .

**Comment by kamleshverma1** [comment]
@gnehapk Added table-view for import cluster .. please review it.

File: src/commons/components/table-view/table-view.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<div class="table-view">
+	<div class="margin-from-top">
+		<h3>{{heading}} chalo aaya</h3>
```
Comment (gnehapk) [inline src/commons/components/table-view/table-view.html]: Please correct the text inside <h3> tag.

**Comment by kamleshverma1** [comment]
abandon

**Pull request closed without merging**